src/parameter_bridge.cpp

```diff
@@ -86,9 +86,15 @@ int main(int argc, char * argv[])
         "with ROS 2 type '%s'\n",
         topic_name.c_str(), type_name.c_str());
 
+      auto ros2_publisher_qos = rclcpp::QoS(rclcpp::KeepLast(queue_size));
+      if (topic_name == "/tf_static") {
```

Comment:
I see how this is an improvement, thought it seems a little fragile. It won't set the QoS settings if the `/tf_static` has been remapped.

Reply:
I share your concern about the `tf_static` being hardcoded but currently I also do not have a better solution. Maybe this could be solved with the approaches that were discussed in the issues that mentioned this PR?